docs/reference/scripting/using.asciidoc

```diff
@@ -155,6 +155,21 @@ POST _scripts/calculate-score
 -----------------------------------
 // TEST[setup:twitter]
 
+Specify a context as part of the url path to test compile a stored script
```

Comment:
Maybe start with "You may also"

Reply:
Also, I would remove "test", since this is just about compiling, it does not run the script.

Reply:
Done.